boa/src/syntax/parser/expression/unary_operator.rs

```diff
@@ -0,0 +1,77 @@
+//! Unary operator parsing.
```

Comment:
Maybe we should call it just `unary` module instead of `unary_expression`. It already is in `expression` module so the expression part is implied.